client/network/src/behaviour.rs

```diff
@@ -203,7 +200,7 @@ pub enum BehaviourOut<B: BlockT> {
 impl<B, Client> Behaviour<B, Client>
 where
 	B: BlockT,
-	Client: HeaderBackend<B> + 'static,
+	Client: HeaderBackend<B> + BlockBackend<B> + 'static,
```

Comment:
Sorry, I should have been more clear on what to revert :see_no_evil: 

This trait should be not needed here. So, could you please revert adding this trait.

Reply:
Sorry that was my oversight, fixed